testing/gpg-remailer/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=gpg-remailer
+pkgver=3.02.01
+pkgrel=0
+pkgdesc="decrypt messages and re-encrypt them to a well-defined group"
+url="https://fbb-git.github.io/gpg-remailer/"
+arch="all"
+license="GPL3+"
+makedepends="icmake bash yodl bobcat bobcat-dev"
```

Comment:
`foo-dev` package should typically depend on `foo`, so `bobcat` should not be needed here, only `bobcat-dev`.